sdkconfig

```diff
@@ -143,14 +146,14 @@ CONFIG_NIMBLE_CPP_ENABLE_ADVERTISMENT_TYPE_TEXT=y
 #
 # Serial flasher config
 #
-CONFIG_ESPTOOLPY_PORT="/dev/cu.usbserial-*"
-CONFIG_ESPTOOLPY_BAUD_115200B=
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+CONFIG_ESPTOOLPY_BAUD_115200B=y
 CONFIG_ESPTOOLPY_BAUD_230400B=
 CONFIG_ESPTOOLPY_BAUD_921600B=
 CONFIG_ESPTOOLPY_BAUD_2MB=
-CONFIG_ESPTOOLPY_BAUD_OTHER=y
-CONFIG_ESPTOOLPY_BAUD_OTHER_VAL=1500000
-CONFIG_ESPTOOLPY_BAUD=1500000
+CONFIG_ESPTOOLPY_BAUD_OTHER=
+CONFIG_ESPTOOLPY_BAUD_OTHER_VAL=115200
+CONFIG_ESPTOOLPY_BAUD=115200
```

Comment:
Really? `1500000` is my favourite! Atom can flash this fast. `115200` is so slow.